sdk/core/azure-core/src/main/java/com/azure/core/credential/TokenRefreshOptions.java

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.credential;
+
+import java.time.Duration;
+
+/**
+ * The options to configure the token refresh behavior.
+ */
+public class TokenRefreshOptions {
+    private static final Duration DEFAULT_REFRESH_RETRY_TIMEOUT = Duration.ofSeconds(30);
+    private static final Duration DEFAULT_REFRESH_OFFSET = Duration.ofMinutes(2);
+
+    private Duration tokenRefreshRetryTimeout = DEFAULT_REFRESH_RETRY_TIMEOUT;
+    private Duration tokenRefreshOffset = DEFAULT_REFRESH_OFFSET;
+
+    /**
+     * Returns a Duration value representing the amount of time to wait before retrying a token refresh. This is to
+     * prevent sending too many requests to the authentication service.
+     *
+     * @return the duration value representing the amount of time to wait before retrying a token refresh
+     */
+    public Duration getTokenRefreshRetryTimeout() {
```

Comment:
Since this is on the `TokenRefreshOptions` should we remove the "TokenRefresh" from these property accessors? In this case `getRetryTimeout`

Reply:
Simplified as suggested